test/Altinn.FileScan.Tests/TestingServices/DataElementServiceTests.cs

```diff
@@ -99,10 +99,10 @@ public async Task Scan_TimestampDoesNotMatch_ErrorLogged()
             // Arrange
             Mock<IAppOwnerBlob> blobMock = new();
             blobMock
-                .Setup(b => b.GetBlobProperties(It.IsAny<string>(), It.IsAny<string>()))
+                .Setup(b => b.GetBlobProperties(It.IsAny<string>(), It.IsAny<string>(), null))
```

Comment:
Fix syntax

Reply:
I'm not sure exactly what happens when we feed it actual values, but it's best to be consistent. It could bite us in the future. Fix all occurences.